include/odp/api/abi-default/packet.h

```diff
@@ -19,6 +19,9 @@ typedef struct { char dummy; /**< @internal Dummy */ } _odp_abi_packet_t;
 /** @internal Dummy  type for strong typing */
 typedef struct { char dummy; /**< *internal Dummy */ } _odp_abi_packet_seg_t;
 
+/** @internal Dummy  type for strong typing */
+typedef struct { char dummy; /**< *internal Dummy */ } _odp_abi_packet_buf_t;
+
```

Comment:
The difference between packet buffer and packet segment, probably, needs to be clarified (explicitly specified) to avoid confusion in implementation.

Reply:
Fixed, documentation added into odp/api/spec/packet.h.